src/locales/strings/enUS.json

```diff
@@ -617,6 +622,9 @@
   "fio_get_fee_err_msg": "There was an issue getting fee. Please try later.",
   "fio_get_reg_info_err_msg": "There was an issue getting registration information.",
   "fio_register_address_err_msg": "There was an issue registering an address. Please try later.",
+  "fio_request_by_fio_address_error_no_address": "Before you can request funds using FIO Address, you must first register your own FIO Address. See Menu.",
```

Comment:
Change to 

"Before you can request funds using FIO, you must first register your own FIO Address. See side menu."

Reply:
changed